src/BuilderPattern/Director/BmwSeries1.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace BuilderPattern.Director
+{
+    using ConcreteBuilder;
+    using Product;
+    using Product.Parts;
+
+    public class BmwSeries1
+    {
+        public BmwSeries1()
```

Comment:
it's a bit weird that BmwSeries1 class has a property called Car... bmwseries1 itself should be a IVehicle maybe?

Reply:
The naming of the property isn't the best :) I change it to Vehicle its more generic.
I didn't use an interface since Vehicle is the object created and I want to expose this object as a property.